tests/ios/frameworks/objc/BUILD.bazel

```diff
@@ -9,6 +9,10 @@ apple_framework(
         "main.m",
     ],
     platforms = {"ios": "10.0"},
+    resource_bundles = {"FooResources": glob(
+        ["Resources/**/*"],
+        exclude_directories = 0,
```

Comment:
I'm a bit confused about the use case here... why do we need `exclude_directories = 0` here? Is this a usage that `rules_apple` works with and `rules_ios` does not?

Reply:
In `cocoapods-bazel`, [this line](https://github.com/bazel-ios/cocoapods-bazel/blob/master/lib/cocoapods/bazel/target.rb#L355) explicitly set `exclude_directories = 0`. I will check if we can set this to `1` without losing any bundle file.